src/js/pages/KanbanBoard/components/UtilityPanel/UtilityPanel.js

```diff
@@ -45,6 +45,8 @@ export default class UtilityPanel extends React.Component {
     const query = QueryString.parse(search);
     takes.map(tk=>{ if (tk.published) { publishedTakeLocation = tk.location;} } );
 
+    let toggled = localStorage.getItem("panelToggled") || "true";
+    
```

Comment:
what was the reasoning behind wanting to store the toggledState of the utilityPanel in local storage?

Reply:
@natibekele This was in user story

> Consider setting the side toggle default to the verse/chunk display rather than the comment system or remember last user state.

So the app saves this preference to the localStorage

Reply:
ok. got it